### PR TITLE
react-forms

### PR DESCRIPTION
**Pull request by MariyaPikalova**

### CODE REVIEW

**Review by iliatcymbal — APPROVED** [review]

File: src/TextEdit/TextEdit.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    const { hidden, text } = this.state;
+    return (
+      <div className="text-edit">
+        { hidden && <span onClick={() => this.setState({ hidden: false })}>{text}</span> }
+        { !hidden &&
+        <input
+          type="text"
+          onBlur={this.handleBlur}
+          defaultValue={text}
+          ref={input => this.handleFocus(input)}
```
Comment (iliatcymbal) [inline src/TextEdit/TextEdit.js:38]: it can be simplier `ref={this.handleFocus}`

File: src/Aside/Aside.js [hunk 2]
```diff
@@ -1,13 +1,22 @@
-import { Numbers } from '../Numbers';
-import { ListOfUsers } from '../ListOfUsers';
+import { Form } from '../Form';
```
Comment (iliatcymbal) [inline src/Aside/Aside.js:1]: where is Form in your repository?

File: src/Todos/todos.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  icon {
+    display: inline-block;
+    margin: 5px 5px;
+    cursor: pointer;
+    vertical-align: middle;
+    height: 21px;
+    width: 21px;
+  }
+  .done {
+    background: url('../images/done.png') no-repeat center;
```
Comment (iliatcymbal) [inline src/Todos/todos.scss:20]: where is images folder?

**Review by iliatcymbal — COMMENTED** [review]
I got lots of errors - some files are not exising. Please, fix it and let me know
http://joxi.ru/DrlVqNJuvPD4O2

**Comment by MariyaPikalova** [comment]
Corrected.

**Pull request merged**